### PR TITLE
[@kadena/transfer] Enable safe transfer

### PR DESCRIPTION
**Pull request by KristinaSpasevska**
## Reason:

Closes [issue link]

## Check off the following:

- [ ] I have reviewed my changes and run the appropriate tests.
- [ ] I have have run `rush change` to add the appropriate change logs.
- [ ] I have added/edited docs.
- [ ] I have added tutorials.
- [ ] I have double checked and DEFINITELY added docs.

### CODE REVIEW

**Comment by nil-amrutlal-dept** [comment]
I've made some changes to the existing variable names and also to the structure of the code on my cross-chain transfer branch. Since I am not able to complete the transfer yet, I'd say we can merge this one when ready and then I can refine everything (there will probably be conflicts).

**Review by nil-amrutlal-dept — APPROVED** [review]

File: packages/apps/transfer/src/services/transfer/coin-transfer.ts
Comment (alber70g) [inline packages/apps/transfer/src/services/transfer/coin-transfer.ts:102]: In the implementation of `convertDecimal` it'd be also better to use either `{ decimal: "10.0" }` or PactNumber or BigNumber

File: packages/apps/transfer/src/services/transfer/coin-transfer.ts
Comment (alber70g) [inline packages/apps/transfer/src/services/transfer/coin-transfer.ts:106]: Is it on purpose that only `x:y` type accounts are allowed? There can be named accounts that don't have the prefix `x:` (`x` being anything)

File: packages/apps/transfer/src/services/transfer/coin-transfer.ts
Comment (alber70g) [inline packages/apps/transfer/src/services/transfer/coin-transfer.ts:111]: Idem

File: packages/apps/transfer/src/services/transfer/coin-transfer.ts
Comment (alber70g) [inline packages/apps/transfer/src/services/transfer/coin-transfer.ts:153]: The sender and receiver could be signed in a separate process where the transaction from `createCommand()` could be sent to the other person via an async process. 
Does the feature allow for that use case?

**Review by alber70g — CHANGES_REQUESTED** [review]
Some comments

**Comment by MRVDH** [comment]
Note: PR on hold since it was decided not to do transfers in the dev tooling website yet.

**Pull request closed without merging**
